react/components/AddressCompletionForm.tsx

```diff
@@ -168,6 +168,7 @@ const AddressCompletionForm: React.FC<Props> = ({
         )}
 
         <Button
+          testId="continue-shipping"
```

Comment:
To keep the test ids consistent, I think that it would be good to change this to `continue-shipping-button`, like here https://github.com/vtex-apps/checkout/pull/65/files#diff-941afac93638c0272183f66e9a469c4bbb5bf5eed1deb5a07c02bcd9a4580a3bR216

OR change the other PR that I've linked to `place-order` (without the `button` suffix)

Was there any pattern before? I don't really remember. Also, if I had to choose between both, I would go with always using the HTML element on suffix (`foo-button`, `bar-qux-input`, and so on).

Reply:
Nice! Done